### PR TITLE
Shortened quilt definitions

### PR DESCRIPTION
**Pull request by T19700**
shortens the definitions found in quilt.fish to around 70 characters or less (one exception at 80). I also fixed spacing issues and made grammar and capitalization consistent.

Worked on issue #6981 

This is my first time working with GitHub in this way. Let me know if any edits need to be made.

### CODE REVIEW

File: share/completions/quilt.fish [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
-complete -c quilt -a grep -r --no-file -d "Grep through the source files, recursively, skipping patches and quilt meta-information"
-complete -c quilt -a header -r --no-file -d " Print or change the header of the topmost or specified patch"
+complete -c quilt -a delete -r -d "Remove the topmost/given patch from the series file"
+complete -c quilt -a diff -r -d "Produces a diff of the given file(s) in the topmost/given patch"
+complete -c quilt -a edit -r -d "Edit the given file(s) in $EDITOR"
+complete -c quilt -a files -r -d "Lists files that the topmost/given patch changes"
+complete -c quilt -a fold -r -d "Integrate the patch read from standard input into the topmost patch"
+complete -c quilt -a fork -r --no-file -d "Fork the topmost patch"
+complete -c quilt -a graph -r --no-file -d "Generates a dot(1) directed graph of dependencies between applied patches"
+complete -c quilt -a grep -r --no-file -d "Grep through source files recursively (skips patches and quilt meta-information)"
```
Comment (faho) [inline share/completions/quilt.fish:17]: We already know that it's source files, no need to list patches and meta-stuff specifically. So just "grep through source recursively"?

File: share/completions/quilt.fish [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+complete -c quilt -a new -r -d "Creates new patch with the given name, inserts it after topmost patch"
+complete -c quilt -a next -r -d "Prints name of patch after the topmost/given patch in the series file"
+complete -c quilt -a patches -r -d "Print the list of patches that modify the given file"
+complete -c quilt -a pop -r -d "Remove patch(es) from the stack of applied patches"
+complete -c quilt -a previous -r -d "Prints name of patch before the topmost/given patch in the series file"
 complete -c quilt -a push -r -d "Apply patch(es) from the series file"
-complete -c quilt -a refresh -r -d "Refreshes the specified patch, or the topmost patch by default"
-complete -c quilt -a remove -r -d "Remove one or more files from the topmost or named patch"
-complete -c quilt -a rename -r -d "Rename the topmost or named patch"
+complete -c quilt -a refresh -r -d "Refreshes the given patch, or the topmost patch by default"
```
Comment (faho) [inline share/completions/quilt.fish:27]: We use the imperative mood - "Refresh the given patch...". This consistently saves characters ("refresh" vs "refreshes")

Comment (faho) [reply share/completions/quilt.fish:27]: (this goes for the rest as well!)

File: share/completions/quilt.fish [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
+complete -c quilt -a remove -r -d "Remove one or more files from the topmost/named patch"
+complete -c quilt -a rename -r -d "Rename the topmost/named patch"
 complete -c quilt -a series -r --no-file -d "Print the names of all patches in the series file"
-complete -c quilt -a setup -r -d " Initializes a source tree from an rpm spec file or a quilt series file"
-complete -c quilt -a snapshot -r --no-file -d " Take a snapshot of the current working state"
-complete -c quilt -a top -d "Print the name of the topmost patch on the current stack of applied patches"
-complete -c quilt -a unapplied -r -d "Print a list of patches that are not applied, or all patches that follow the specified patch in the series file"
-complete -c quilt -a upgrade -d "Upgrade the meta-data in a working tree from an old version of quilt to the current version"
+complete -c quilt -a setup -r -d "Initializes a source tree from an rpm spec file or a quilt series file"
```
Comment (faho) [inline share/completions/quilt.fish:31]: Maybe remove the first "file"? "Initialize a source tree from an rpm spec or quilt service file"?

**Review by faho — CHANGES_REQUESTED** [review]

File: share/completions/quilt.fish [hunk 4]
```diff
@@ -1,35 +1,35 @@
 # completions for quilt version 0.46 or similar
-complete -c quilt -s h -d "show help"
-complete -c quilt -l version -d "show version"
-complete -c quilt -l quiltrc -r --no-file -d "specify the file to Run Control file to use"
+complete -c quilt -s h -d "Show help"
+complete -c quilt -l version -d "Show version"
+complete -c quilt -l quiltrc -r --no-file -d "Specify the file to Run Control file to use"
```
[the remark this answers is not in the document]
Comment (T19700) [reply share/completions/quilt.fish:4]: would you like me to remove the "--no-file"?

[the remark this answers is not in the document]
Comment (faho) [reply share/completions/quilt.fish]: Unless you know that it's not needed, not right now.

I'm just musing that we should take a look at it, but until someone does that I'd always trust the initial author, as they typically know the thing the completions are for (that's why they wrote them!).

[the remark this answers is not in the document]
Comment (T19700) [reply share/completions/quilt.fish]: I fixed the rc definition, just let me know about the "--no-file" and ill fix that if you want me to!

**Comment by faho** [comment]
Merged, sorry for the delay!

**Pull request merged**